app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -191,6 +193,10 @@ snippet: true
     done
   <% end %>
 
+  <% if host_param_true?('subscription_manager_auto_attach', false) %>
+    subscription-manager attach --auto
+  <% end %>
+
```

Comment:
I suspect there will be differences in the snapshot but you can avoid that I think:
```suggestion

  <% end -%>
```

This will not add the whitespace changes.